src/components/date_picker/date_picker.tsx

```diff
@@ -114,9 +151,12 @@ interface EuiExtendedDatePickerProps
   iconType?: EuiFormControlLayoutIconsProps['icon'];
 
   /**
-   * Sets the placement of the popover
+   * Sets the placement of the popover.
+   *
+   * DEPRECATED: 'bottom', 'bottom-end', 'bottom-start', 'left', 'left-end', 'left-start', right', 'right-end', 'right-start', 'top', 'top-end', 'top-start'
+   * Use `EuiPopover` values instead: 'upCenter', 'upLeft', 'upRight', downCenter', 'downLeft', 'downRight', 'leftCenter', 'leftUp', 'leftDown', 'rightCenter', 'rightUp', 'rightDown',
```

Comment:
```suggestion
   *
   * **Use [EuiPopover](/#/layout/popover) values**: 'upCenter', 'upLeft', 'upRight', downCenter', 'downLeft', 'downRight', 'leftCenter', 'leftUp', 'leftDown', 'rightCenter', 'rightUp', 'rightDown'.
```

Just makes it more findable.

Reply:
Maybe also change the default to `downLeft`?